pages/restmail.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import time
+import requests
+
+
+def get_login_link(username, message_count=1, timeout=60):
+    username = username.partition('@restmail.net')[0]
+    end_time = time.time() + timeout
+    while (True):
+        response = requests.get(
+            'http://restmail.net/mail/%s' % username,
+            verify=False)
```

Comment:
Also it's a good practice to raise an error if the status code of the HTTP response is not 200. ``requests`` provides this helper:

http://docs.python-requests.org/en/master/api/#requests.Response.raise_for_status